tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java

```diff
@@ -215,6 +225,33 @@ public void testSiteUseCasesForCreateModifyDeleteSite()
         assertSiteQueryResult(testSite1.getId(), List.of());
         assertSiteQueryResult(ALL_SITES, List.of());
         assertSiteQueryResult(EVERYTHING, "AND", FILE_CONTENT_CONDITION, List.of(fileNotInSite));
+
+        Step.STEP("Moving files between sites and modifying content use cases");
```

Comment:
I guess the tests from this line onwards could either be moved to their own method. If not, then would you mind renaming the existing method?

Reply:
Do you mean that I should create a separate test method for my cases? or a method within your test method?

Reply:
Either a separate test method for your cases, or rename the method from "testSiteUseCasesForCreateModifyDeleteSite()" to something that is more generic or covers document use cases as well.

Reply:
moved to separate test method